Makefile.am

```diff
@@ -73,7 +73,7 @@ noinst_HEADERS += examples/random.h
 PRECOMPUTED_LIB = libsecp256k1_precomputed.la
 noinst_LTLIBRARIES = $(PRECOMPUTED_LIB)
 libsecp256k1_precomputed_la_SOURCES =  src/precomputed_ecmult.c src/precomputed_ecmult_gen.c
-libsecp256k1_precomputed_la_CPPFLAGS = $(SECP_INCLUDES)
+libsecp256k1_precomputed_la_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src $(SECP_INCLUDES)
```

Comment:
```suggestion
# We need `-I$(top_srcdir)/src` in VPATH builds if libsecp256k1_precomputed_la_SOURCES have been cleaned from the source tree and then recreated in the build tree.
# This helps users and packagers who insist on recreating the precomputed files (e.g., Gentoo).
libsecp256k1_precomputed_la_CPPFLAGS = -I$(top_srcdir)/src $(SECP_INCLUDES)
```

`-I$(top_srcdir)/src` suffices. We use relative includes throughout the library, and since `src/precomputed_ecmult*.c` believe they live in `-I$(top_srcdir)/src`, everything they include is reachable from there.

Reply:
> _libsecp256k1_precomputed_la_SOURCES have been cleaned from the source tree_

This is not required to trigger the error. Having them in the build tree is enough for it.

Therefore, the comment could be adjusted.

Reply:
> This is not required to trigger the error. Having them in the build tree is enough for it.

Hm strange. What are the exact steps you've run to reproduce this?

Reply:
> Hm strange. What are the exact steps you've run to reproduce this?

```
$ ./autogen.sh
$ mkdir ../build && cd ../build
$ ../secp256k1/configure
$ make precompute_ecmult precompute_ecmult_gen
$ ./precompute_ecmult
$ ./precompute_ecmult_gen
$ make
```

Reply:
Oh indeed, I edited my suggestion.